src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -359,7 +380,7 @@ private static void handleCloningFailed(List<RepoConfiguration> configs, RepoLoc
      * removes {@code failedConfig} from the list of {@code configs}.
      */
     private static void handleAnalysisFailed(List<RepoConfiguration> configs, RepoConfiguration failedConfig,
-            String errorMessage) {
+                                             String errorMessage) {
```

Comment:
I think the indentation is incorrect.

Reply:
Is it possible to add a checkstyle rule to ensure this indentation rule is followed instead of manually warning about it in the reviews?

Reply:
I have fixed the indentation here.

Reply:
I believe it is better to open a separate PR for this

Reply:
As pointed out, enforcing this checkstyle rule will violate the coding standard of `Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line.` I think it's best to leave it for now and focus on non trivial aspects of this issue.